internal/proxy/proxy.go

```diff
@@ -134,6 +141,8 @@ func (p *Proxy) Handle(w http.ResponseWriter, r *http.Request) {
 		requestType = "Jenkins UI"
 	}
 
+	Recorder.RecordReqByTypeTotal(requestType)
```

Comment:
It would be good if you can move metric recording to some middlewares.  Similar (not exactly) to https://github.com/fabric8-services/fabric8-wit/blob/master/metric/recorder.go#L20.  So logic of metrics recording will be separated from actual request handling logic which consider clean design and it has it's benefits.

Reply:
Also, I see that prometheus is registered at - https://github.com/fabric8-services/fabric8-jenkins-proxy/blob/master/internal/router/router.go#L14
Actually, prometheus.Handler() is deprecated.  If possible, use promhttp.Handler().  Check - https://github.com/fabric8-services/fabric8-wit/blob/master/main.go#L416.